### ISSUE TITLE
`stern` process leak in Makefile?

### ISSUE THREAD

**Issue by simster7** [argoproj/argo-workflows#7359]
## Summary

What happened/what you expected to happen?

`htop` sorted by highest virtual memory
<img width="895" alt="image" src="https://user-images.githubusercontent.com/614838/145252328-76682582-4d64-4b0f-91fc-31d386e8c30f.png">

What version of Argo Workflows are you running?

## Diagnostics

Either a workflow that reproduces the bug, or paste you whole workflow YAML, including status, something like:

```yaml
kubectl get wf -o yaml ${workflow}
```

What Kubernetes provider are you using?

What executor are you running? Docker/K8SAPI/Kubelet/PNS/Emissary

```bash
# Logs from the workflow controller:
kubectl logs -n argo deploy/workflow-controller | grep ${workflow}

# The workflow's pods that are problematic:
kubectl get pod -o yaml -l workflows.argoproj.io/workflow=${workflow},workflow.argoproj.io/phase!=Succeeded

# Logs from in your workflow's wait container, something like:
kubectl logs -c wait -l workflows.argoproj.io/workflow=${workflow},workflow.argoproj.io/phase!=Succeeded
```

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by simster7** [comment]
Workaround: `killall -9 stern`

**Comment by whynowy** [comment]
> Workaround: `killall -9 stern`

Could you add it to `Makefile`?

**Comment by agilgur5** [comment]
This was potentially fixed by #8634 / #8613
